src/lib/db-listeners/db-other-docs-listener.ts

```diff
@@ -97,6 +97,7 @@ export class DBOtherDocumentsListener extends BaseListener {
       const documentModel = documents.getDocument(dbDoc.self.documentKey);
       if (documentModel) {
         documentModel.setTitle(dbDoc.title);
+        if (dbDoc.visibility) documentModel.setVisibility(dbDoc.visibility);
```

Comment:
This seems like it might not work right. If the visibility of the `dbDoc` is false or undefined, then the visibility of the documentModel will not be set to false or undefined.

Reply:
> This seems like it might not work right. If the visibility of the `dbDoc` is false or undefined, then the visibility of the documentModel will not be set to false or undefined.

I had the same thought initially, but since the valid values for `visibility` are `public` and `private`, that may well be the correct behavior.